src/base.html

```diff
@@ -145,13 +145,15 @@
             href="https://fonts.googleapis.com/css?family={{
               font.text | replace(' ', '+') + ':300,400,400i,700%7C' +
               font.code | replace(' ', '+')
-            }}&display=fallback" />
+            }}&display=swap" />
         <style>
           body, input {
+            font-display: swap;
             font-family: "{{ font.text }}", "Helvetica Neue",
               Helvetica, Arial, sans-serif;
           }
           pre, code, kbd {
+            font-display: swap;
```

Comment:
I don't think this needs to be defined in the style block here... the `font-display:` is just for the @font-face rule

Reply:
Correct, it's [a descriptor](https://www.w3.org/TR/css-fonts-4/#font-display-desc). We should remove it from here.